tests/Dom_2/dom.js

```diff
@@ -0,0 +1,53 @@
+const file_path = process.env.FILE_PATH;
+const webdriver = require('selenium-webdriver');
+const assert = require('chai').assert;
+const { By, Actions } = require("selenium-webdriver");
+
+
+describe("Testing Scholars Project", function () {
+    this.timeout(3 * 1000 * 60);
+    var driver = new webdriver.Builder()
+
+        .forBrowser('chrome')
+        .build();
+
+    var actions = driver.actions();
+
+    before(function setupWebdriver(done) {
+        var file_url = `file://${file_path}`;
+        driver.get(file_url).then(done);
+    });
+
+    after(function () {
+        driver.quit();
+    });
+
+    it('test case: check dot on Cart', async () => {
+        const click = await driver.findElements(By.css("button"));
+        click[2].click();
+        const option = await driver.findElement(By.className("badge")).isDisplayed();
+        assert.exists(option == true, "Ensure the badge/dot is present on Cart")
+    });
+
+    it('test case: check text on button', async () => {
+        const click = await driver.findElements(By.css("button"));
+        click[2].click();
+        const button_text = await click[2].getText();
+        assert.equal(button_text, "Remove", 'Ensure the text on the button changes to `Remove` after adding to Cart`')
+    });
+
+    it('test case: check colour of text', async () => {
+        const click = await driver.findElements(By.css("button"));
+        click[2].click();
+        const text_color = await click[2].getCssValue("color");
+        assert.equal(text_color, "rgba(78, 77, 77, 1)", 'Ensure the colour of the text `Remove` is white')
```

Comment:
I think the same thing with the class issue. There is no way to tell which type of color scheme they are using from `rgb`, `rgba`,  or `hex`

Reply:
We expect the text value to be white. Selenium will interprete whatever colour scheme they use into rgba. Just noticed I didn't use the right rgba colour scheme for white. I will update it.